compile_proto_wrappers.sh

```diff
@@ -21,4 +21,6 @@ protoc \
   --python_out=. \
   --pyi_out=. \
   --js_out=import_style=commonjs,binary:js \
-  ../ord-schema/proto/*.proto
+  ../ord-schema/proto/*.proto \
+  --plugin="protoc-gen-ts=./js/ord-schema/node_modules/.bin/protoc-gen-ts" \
```

Comment:
If you can, please update this to remove `--plugin` and assume that protoc-gen-ts is in the path, like we do with JS: https://github.com/open-reaction-database/ord-schema/blob/main/.github/workflows/run_tests.yml#L88 (and update the GitHub workflow to add it)

Reply:
Thanks a lot for the explanation, I will update the PR tomorrow then